modules/core/src/rand.cpp

```diff
@@ -60,6 +60,10 @@
     #include "emmintrin.h"
 #endif
 
+#if defined __ARM_NEON && defined __aarch64__
+    #include <arm_neon.h>
+#endif
```

Comment:
It should be handled by [this code](https://github.com/opencv/opencv/blob/4.5.1/modules/core/include/opencv2/core/cv_cpu_dispatch.h#L136-L143).

Could you please add dump of error message and passed compiler options?